vhdbuilder/packer/windows/components_json_helpers.tests.ps1

```diff
@@ -807,6 +807,8 @@ Describe 'Tests of components.json ' {
         #        $packages["c:\akse-cache\win-k8s\"] | Should -Contain "https://acs-mirror.azureedge.net/kubernetes/v1.31.3/windowszip/v1.31.3-1int.zip"
         $packages["c:\akse-cache\win-k8s\"] | Should -Contain "https://acs-mirror.azureedge.net/kubernetes/v1.31.4/windowszip/v1.31.4-1int.zip"
         $packages["c:\akse-cache\win-k8s\"] | Should -Contain "https://acs-mirror.azureedge.net/kubernetes/v1.31.5/windowszip/v1.31.5-1int.zip"
+        $packages["c:\akse-cache\win-k8s\"] | Should -Contain "https://acs-mirror.azureedge.net/kubernetes/v1.32.0/windowszip/v1.32.0-1int.zip"

```

Comment:
I've got a PR to modify the way were testing these on Windows, so this change shouldn't be necessary once that is merged. There's a risk that if my PR merges before yours, then we might have a test failure in master as a result. I'll monitor that tho.

Reply:
As you saied ,it cause a test failure, should I remove this change?

Reply:
Yup - backing the change out is the easiest fix.